lib/src/solver/report.dart

```diff
@@ -434,8 +426,11 @@ $contentHashesDocumentationUrl
     }
 
     // Highlight overridden packages.
-    if (isOverridden && highlightOverride) {
-      output.write(" ${log.magenta('(overridden)')}");
+    if (isOverridden) {
+      final location = _root.pubspec.usesOverridesFile
+          ? ' in ${p.join(_root.dir, Pubspec.pubspecOverridesFilename)}'
+          : '';
```

Comment:
Shouldn't we only write this for packages are overridden in the pubspec_overrides.yaml file?

Reply:
overrides can only all come from the pubspec_overrides.yaml file or none of them